### PR TITLE
Change default results_timeout to 12h

### PR DESCRIPTION
**Pull request by zmc**
Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

**Comment by yuriw** [comment]
Cool, looks good, thx !

**Comment by andrewschoen** [comment]
LGTM

File: docs/siteconfig.rst [hunk 1]
```diff
@@ -40,6 +40,10 @@ Here is a sample configuration with many of the options set and documented::
     # Email address that job results summaries originate from
     results_sending_email: teuthology@example.com
 
+    # How long teuthology-results should wait for jobs to finish before
+    # considering them 'hung'
+    results_timeout: 43200
```
Comment (andrewschoen) [inline docs/siteconfig.rst]: this in seconds?

Comment (zmc) [inline docs/siteconfig.rst]: ah, yes it is. I can add a note clarifying that.

**Pull request merged**